src/Product/ProductAttribute.php

```diff
@@ -13,14 +13,20 @@ class ProductAttribute
      */
     private $value;
 
+    /**
+     * @var string
+     */
+    private $type;
+
     /**
      * @param string $name
      * @param string $value
      */
-    public function __construct($name, $value)
+    public function __construct($name, $value, $type)
```

Comment:
```suggestion
    public function __construct($name, $value, $type = '')
```

I'm adding this so that I can include it in a patch / minor release, otherwise this would constitue a breaking change.